test/integration/user_test.rb

```diff
@@ -19,8 +19,8 @@ class UserIntegrationTest < IntegrationTestWithJavascript
   context "without automatic login" do
     def login_admin
     end
-
     test "login" do
+      Capybara.current_driver = Capybara.javascript_driver
```

Comment:
any idea why is this needed?

Reply:
It seem that without it the test can't mount the react component,
therefor can't find the right input element.

Reply:
using the `login_user` helper instead (as we discussed offline)